cmake/kokkos_install.cmake

```diff
@@ -32,10 +32,11 @@ IF (NOT KOKKOS_HAS_TRILINOS AND NOT Kokkos_INSTALL_TESTING)
 
   # Required to be a TriBITS-compliant external package
   file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos)
-  file(CREATE_LINK ${Kokkos_BINARY_DIR}/KokkosTargets.cmake ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos/KokkosTargets.cmake SYMBOLIC)
-  file(CREATE_LINK ${Kokkos_BINARY_DIR}/KokkosConfig.cmake ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos/KokkosConfig.cmake SYMBOLIC)
-  file(CREATE_LINK ${Kokkos_BINARY_DIR}/KokkosConfigCommon.cmake ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos/KokkosConfigCommon.cmake SYMBOLIC)
-  file(CREATE_LINK ${Kokkos_BINARY_DIR}/KokkosConfigVersion.cmake ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos/KokkosConfigVersion.cmake SYMBOLIC)
+  file(COPY ${Kokkos_BINARY_DIR}/KokkosConfig.cmake
+            ${Kokkos_BINARY_DIR}/KokkosConfigCommon.cmake
+            ${Kokkos_BINARY_DIR}/KokkosConfigVersion.cmake 
+            DESTINATION ${CMAKE_BINARY_DIR}/cmake_packages/Kokkos)
```

Comment:
Why is this not an `install` command?

Reply:
because it doesn't go in the install directory.